src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -176,7 +176,7 @@ static bool IsKeywordItem(CompletionItem item)
                 => item.Tags.Contains(WellKnownTags.Keyword);
 
             static bool TagsEqual(CompletionItem item1, CompletionItem item2)
-                => item1.Tags == item2.Tags || item1.Tags.SequenceEqual(item2.Tags, arg: true, static (a, b, _) => a == b);
+                => System.Linq.ImmutableArrayExtensions.SequenceEqual(item1.Tags, item2.Tags);
```

Comment:
does this not work with `using System.Linq;`?

Reply:
It should, I made it fully qualified to visually differentiate it from other helpers we have with the identical name